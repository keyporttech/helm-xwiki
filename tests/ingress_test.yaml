suite: test ingress
templates:
  - ingress.yaml
  - service.yaml
tests:
  - it: should set values properly
    set:
      annotations.kubernetes.io/ingress.class: nginx
      service.type: ClusterIP
      service.externalPort: 80
      service.internalPort: 8080
      service.name: tcp
      ingress.enabled: true
    release:
      name: release
    asserts:
      - isKind:
          of: Ingress
      - matchSnapshot:
          path: metadata.annotations
      - equal:
          path: metadata.name
          value: release-xwiki
      - matchRegex:
          path: metadata.labels.chart
          pattern: xwiki-*
      - matchSnapshot:
          path: spec