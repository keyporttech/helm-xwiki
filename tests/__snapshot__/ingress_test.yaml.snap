should set values properly:
  1: |
    kubernetes.io/ingress.class: nginx
  2: |
    rules:
    - http:
        paths:
        - backend:
            serviceName: release-xwiki
            servicePort: 80
          path: /
